sdk/security/keyvault/azcertificates/MIGRATION.md

```diff
@@ -0,0 +1,96 @@
+# Guide to migrate from `keyvault` and to `azcertificates`
+
+This guide is intended to assist in the migration to the `azcertificates` module from the deprecated `keyvault` module. `azcertificates` allows users to create and manage [certificates][certificates] with Azure Key Vault.
+
+## General changes
+
+In the past, Azure Key Vault operations were all contained in a single package. For Go, this was `github.com/Azure/azure-sdk-for-go/services/keyvault/<version>/keyvault`. 
+
+The current strategy is to break up the Key Vault into separate modules by functionality. Now there is a specific module for keys, secrets, and certificates. This guide focuses on migrating certificate operations to use the new `azcertificates` module.
+
+Besides, module name changes. There are a number of name differences for methods and variables. All new modules also authenticate using our [azidentity] module.
+
+## Code examples
+
+### `keyvault` create certificate
```

Comment:
We should point out the before/after element here i.e., we have an example using the old SDK followed by one using the new SDK